src/OrbitQt/TargetLabel.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_QT_TARGET_LABEL_H_
+#define ORBIT_QT_TARGET_LABEL_H_
+
+#include <QLabel>
+
+#include "OrbitClientData/ProcessData.h"
+#include "OrbitGgp/Instance.h"
+
+namespace orbit_qt {
+
+class TargetLabel : public QLabel {
+ public:
+  explicit TargetLabel(QWidget* parent = nullptr);
+  void SetFileTarget(const std::filesystem::path& path);
+  void SetStadiaTarget(const ProcessData& process, const orbit_ggp::Instance& instance);
+  void SetLocalTarget(const ProcessData& process);
```

Comment:
I'm not super happy with the API here - there's 3 methods that look like regular setters, but their usage is mutually exclusive.
Maybe just rename them to make this clear, e.g. `ChangeToXXXTarget`.

Reply:
I changed it, ptal